### PR TITLE
Example of publishing packages

### PR DESCRIPTION
**Pull request by scinos**
This is a quick PR to show how publishing packages could be implemented.

I tested that after changing `@kaoto-next/ui`, committing the changes and running `./node_modules/.bin/lerna publish` from the root, it will publish a new version of `@kaoto-next/ui` and `@kaoto-next/ui-tests`

### CODE REVIEW

File: .yarnrc.yml [hunk 1]
```diff
@@ -2,17 +2,26 @@ nodeLinker: node-modules
...
+      'react-dom': '^18.2.0'
+
+npmRegistryServer: 'http://localhost:4873'
+unsafeHttpWhitelist:
+  - 'localhost'
+
+npmScopes:
+  kaoto-next:
+    npmRegistryServer: http://localhost:4873
+    npmPublishRegistry: http://localhost:4873
```
Comment (scinos) [inline .yarnrc.yml:27]: This can be ignored, it is just to test the changes with a local NPM registry.

File: lerna.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    "publish": {
+      // Check devDependencies as well for changed packages
+      "graphType": "all",
+
+      // Use commit prefix to determine minor vs major
+      "conventionalCommits": true,
+
+      "message": "chore(release): publish new version %v",
+
+      "registry": "http://localhost:4873"
```
Comment (scinos) [inline lerna.json:15]: Not needed if publishing to the regular NPM registry.

File: packages/camel-catalog/CHANGELOG.md
Comment (scinos) [inline packages/camel-catalog/CHANGELOG.md:1]: lerna-lite/publish generates this and keeps it up to date based on the commits. Can be tweaked via `--changelog-*` options from https://github.com/lerna-lite/lerna-lite/tree/main/packages/version#options

**Comment by lordrip** [comment]
Thanks a lot for putting this together @scinos :clap: 

This PR relates to: https://github.com/KaotoIO/kaoto-next/issues/77

**Comment by lordrip** [comment]
Thanks a lot @scinos , this pull request will be implemented here: https://github.com/KaotoIO/kaoto-next/pull/173.

**Pull request closed without merging**